Datahub.Web/Env.cs

```diff
@@ -0,0 +1,22 @@
+
+using System;
+
+namespace Datahub.Web
+{
+    public interface IEnv
+    {
+        string ES_INDEX { get; }
+        string ES_SITE { get; }
+    }
+
+    public class Env : IEnv
+    {
+        public string ES_INDEX { get; private set; }
+        public string ES_SITE => "datahub"; // this doesn't really ever change except perhaps for occassional dev use
```

Comment:
If you can see a reason that it could possibly be changed, then just make it a config variable

Reply:
I figured the overhead of having to configure an env variable that isn't really an env variable was net negative. There's no benefit for this being an env var; we don't want to allow the search in any non-dev environment to be against any other site. I'll make it a static readonly value and remove it from the Env class, since it's not really an env var.

Reply:
Done in 8e8a732 👍